packages/contracts/contracts/ArgoBridgeV1.sol

```diff
@@ -188,7 +190,7 @@ contract ArgoBridgeV1 is AccessControl {
 
     function _updateMintingPeriod() internal {
         if (block.number >= currentMintingPeriodEndBlock) {
-            currentMintingPeriodEndBlock = block.number + mintingLimitPeriodLengthBlocks;
+            currentMintingPeriodEndBlock = block.number.add(mintingLimitPeriodLengthBlocks);
```

Comment:
Solidity 0.8 includes SafeMath at language level so I don't believe this makes any difference